rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1971,7 +1974,13 @@ def test_upgrade_db_40_to_41(user_data_dir):
         msg_aggregator=msg_aggregator,
         resume_from_backup=False,
     )
+    # add some settings that should move in the upgrade
+    settings = {
+        name: str(randint(0, 1000)) for name in SETTINGS_MOVED
+        if name is not DBCache.LAST_WRITE_TS.serialize()
+    }
```

Comment:
instead of doing this using structures from the version hardcode them. the reason is that if for some reason we change the logic the test won't be valid anymore